src/main/resources/templates/cypher/form.html

```diff
@@ -1,118 +1,139 @@
 <!DOCTYPE html>
 <html lang="cz"
```

Comment:
Jen drobnost, můžeš ještě změniz cz na cs? To je chybka, kterou jsem tam zavedl kdysi na začátku já, ale cz není validní hodnota.

Reply:
opraveno